src/utils.ts

```diff
@@ -189,3 +189,24 @@ export const calcApr = (
     .div(periodsElapsed)
     .toString();
 };
+/**
+ * Takes two values and returns a list of number intervals
```

Comment:
nit: this explanation is a bit confusing. What is the expected use case for this? In the example below, I don't see an obvious relationship between the inputs and outputs. Maybe a real-world example would help?

Reply:
https://github.com/across-protocol/sdk-v2/pull/6#discussion_r837369366